src/Module/Api/Mastodon/Timelines/PublicTimeline.php

```diff
@@ -90,11 +90,6 @@ protected function rawContent(array $request = [])
 			$condition = DBA::mergeConditions($condition, ['gravity' => Item::GRAVITY_PARENT]);
 		}
 
-		if (!empty($uid)) {
-			$condition = DBA::mergeConditions($condition,
-				["NOT `parent-author-id` IN (SELECT `cid` FROM `user-contact` WHERE `uid` = ? AND (`blocked` OR `ignored`) AND `cid` = `parent-author-id`)", $uid]);
-		}
-
```

Comment:
I drilled down into selectPostsForUser and it ultimately goes to the same View query code that filters out blocked/ignored as well so this seemed duplicate to me even though it was in the code I started with when making this change.